DFC.App.FindACourse/Views/Details/courseDetailsClassroom.cshtml

```diff
@@ -80,49 +80,49 @@
     @if (!string.IsNullOrWhiteSpace(Model.CourseDetails.Description))
     {
         <h2 class="govuk-heading-m" id="whofor">Who this course is for</h2>
-        <p id="course-description">@Html.Raw(Model.CourseDetails.Description)</p>
+        <p id="course-description">@Html.Raw(@Html.HtmlFormat(Model.CourseDetails.Description))</p>
```

Comment:
For this pattern here and below do not think you need the second embedded @ is the following should be ok:
<p id="course-description">@Html.Raw(Html.HtmlFormat(Model.CourseDetails.Description))</p>

Reply:
removed unrequired @